js/interface.js

```diff
@@ -58,7 +59,16 @@ function getDataSources() {
   $dataSources = $('#data-sources > tbody');
 
   Fliplet.DataSources.get().then(function (dataSources) {
```

Comment:
just checked, we do. https://github.com/Fliplet/fliplet-api/blob/master/routes/v1/data-sources.js#L78